applications/main/nfc/helpers/protocol_support/mf_ultralight/mf_ultralight.c

```diff
@@ -185,6 +197,7 @@ const NfcProtocolSupportBase nfc_protocol_support_mf_ultralight = {
             .on_enter = nfc_scene_info_on_enter_mf_ultralight,
             .on_event = nfc_scene_info_on_event_mf_ultralight,
         },
+    .scene_card_dump = {.on_enter = nfc_scene_card_dump_on_enter_mf_ultralight, .on_event = NULL},
```

Comment:
Add common: `.on_event = NULL,` and run linter

Reply:
Done